django/thunderstore/core/settings.py

```diff
@@ -72,6 +73,7 @@
     raise RuntimeError("Could not locate manage.py")
 
 DEBUG = env.bool("DEBUG")
+TESTING = env.bool("TESTING")
```

Comment:
Rather than using a new setting, you can do
```python
TESTING = bool("PYTEST_CURRENT_TEST" in os.environ)
```
as that variable is set by pytest

Reply:
I was looking for something like this but couldn't actually find anything

Reply:
It seems this only works during the test, not in `settings.py`. I'm assuming this is due to the django plugin running imports before it's set